executor/habitat.go

```diff
@@ -43,6 +97,14 @@ func (h *Habitat) exec() (err error) {
 func (h *Habitat) Run() (err error) {
 	lgr.Debug.Println("start installing habitat command.")
 
+	if h.Spec.Habitat.Mode == "local" && h.isDownloaded() == false {
```

Comment:
We should add a debug log of downloading.

Reply:
added.